packages/common/src/core/schemas/hubproject.ts

```diff
@@ -245,5 +251,20 @@ export const HubProjectEditUiSchema: IUiSchema = {
         },
       ],
     },
+    {
+      type: "Section",
+      labelKey: "{{i18nScope}}.relatedItems.label",
+      elements: [
+        {
+          scope: "/properties/relatedItems",
```

Comment:
since this is nested under `view`, the scope will need to be `"/properties/view/properties/relatedItemIds"`